debian/control

```diff
@@ -278,6 +278,7 @@ Package: ceph-mgr-rook
 Architecture: all
 Depends: ceph-mgr (= ${binary:Version}),
          python-six,
+         python-jsonpatch
```

Comment:
i think we should rely on `dh_python2` to do this for us. it is supposed to substitute `${python:Depends}` with the calculated dependencies. could you update the `override_dh_python2` target in `debian/rules` to run `dh_python2` against `ceph-mgr-rook` as well in a different commit and drop this change?

Reply:
done